wgpu/src/renderer.rs

```diff
@@ -222,11 +214,13 @@ impl Renderer {
                 border_width,
                 border_color,
             } => {
-                // TODO: Move some of this computations to the GPU (?)
+                let layer = layers.last_mut().unwrap();
+
+                // TODO: Move some of these computations to the GPU (?)
                 layer.quads.push(Quad {
                     position: [
-                        bounds.x - layer.offset.x as f32,
-                        bounds.y - layer.offset.y as f32,
+                        bounds.x + translation.x,
+                        bounds.y + translation.y,
```

Comment:
We were already applying `layer.offset` manually anyways for most primitives, except `Image` and `Svg`.

I think this is nicer. We also remove a bunch of conversions, so we may be saving some cycles.